pilot/pkg/networking/core/v1alpha3/envoyfilter/rc_patch_test.go

```diff
@@ -524,6 +556,40 @@ func TestApplyRouteConfigurationPatches(t *testing.T) {
 			},
 		},
 	}
+	sidecarInboundRCToBeReplaced := &route.RouteConfiguration{
+		Name: "inbound|http|80",
+		VirtualHosts: []*route.VirtualHost{
+			{
```

Comment:
Please include configuration elements in the to-be-replaced vhost.
I am not sure if the present test case will fail if you just change the operation to merge instead of replace.

Reply:
Thanks for your review :)
> Please include configuration elements in the to-be-replaced vhost that you do not expect in the output.

Done, and I move this to a separate test to avoid confilcts